docs/topics/ref_using-github-labels.adoc

```diff
@@ -1,12 +1,12 @@
 [id='using-github-labels_{context}']
```

Comment:
Out of the scope of this PR obviously, but you no longer use GitHub issues, right? :-)

Reply:
We indeed don't, there's a JIRA filed for that, but since our process isn't 100% fleshed out, I haven't touched the file yet.